doc-Installing_on_OpenShift_Container_Platform/topics/post-installation.adoc

```diff
@@ -145,6 +145,112 @@ $ oc create -f cfme-restore-job.yaml
 
+
+
+[[updating-pods]]
+=== Updating {product-title_short}
+
+To apply package updates to a {product-title_short} appliance running on OpenShift Container Platform, delete the pods and redeploy them using updated {product-title_short} container images. To update your environment with a new template, follow the steps in xref:modified-template-update[].
+
+ifdef::cfme[]
+See the https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html/release_notes/index#technical_notes[Technical Notes] in the _Release Notes_ for a list of updated {product-title_short} container images and errata.
+endif::cfme[]
+
+
+[[image-update]]
+==== Updating the {product-title_short} Container Images
+
+To deploy new container images for {product-title_short}, delete the pods, update the images, then redeploy the updated pods:
+
+. Scale the application StatefulSets to 0 replicas:
++
+----
+$ oc scale statefulset cloudforms --replicas=0
+$ oc scale statefulset cloudforms-backend --replicas=0
+----
++
+. Patch in the new version of the images:
++
+----
+$ oc patch statefulset cloudforms -p '{"spec": {"template": {"spec": {"containers": [{"name": "cloudforms", "image": "registry.redhat.io/cloudforms46/cfme-openshift-app-ui:<new-version-tag>"}]}}}}'
+$ oc patch statefulset cloudforms-backend -p '{"spec": {"template": {"spec": {"containers": [{"name": "cloudforms", "image": "registry.redhat.io/cloudforms46/cfme-openshift-app:<new-version-tag>"}]}}}}'
+----
++
+. Scale the StatefulSets back to your previous values to redeploy the pods:
++
+----
+$ oc scale statefulset cloudforms --replicas=<old-value>
+$ oc scale statefulset cloudforms-backend --replicas=<old-value>
+----
+
+Your {product-title_short} environment is now updated to use the new container images.
+
+
+[[modified-template-update]]
+==== Updating the {product-title_short} Container Images and Template
+
+To update a {product-title_short} deployment with a new template, update the container images and the template using the following steps:
+
+. Back up secrets, such as database encryption keys and credentials:
++
+[IMPORTANT]
+====
+Be careful to back up secrets in a secure location.
+====
++
+----
+$ oc export secret cloudforms-secrets > my_secrets.yml
+----
++
+. Scale the application StatefulSets to 0 replicas:
++
+----
+$ oc scale statefulset cloudforms --replicas=0
+$ oc scale statefulset cloudforms-backend --replicas=0
+----
++
+. Apply the changes to the project, specifying the template appropriate to your environment's database configuration.
++
+[IMPORTANT]
+====
+If you customized any parameters when originally deploying the application (parameters used with the `oc new-app` command in xref:deploying-the-appliance[]), you must set the same values in the `oc process` command here.
+====
++
+Use the following command to apply changes to the project if your environment uses a database stored in a pod within the cluster (the default configuration):
```

Comment:
Here do think there's value in identifying the database type first?

Ex.

1. For Environments using databases stored within a cluster:
 
Do x

2. Databases external to the OCP Cluster:

Do Y

This came up in a recent minimalism session, hence applying it here.

Reply:
Excellent idea -- that is a much better/straightforward way than what I'd written before. Let me know what you think of the new wording.